pdagent/thirdparty/argparse.py

```diff
@@ -1775,7 +1774,7 @@ def parse_known_args(self, args=None, namespace=None):
                 if not hasattr(namespace, action.dest):
                     if action.default is not SUPPRESS:
                         default = action.default
-                        if isinstance(action.default, basestring):
```

Comment:
Meant to leave this as `basestring` for compatibility between versions -- can fix in separate pass.